### ISSUE TITLE
Could not load type 'NReadability.NReadabilityWebTranscoder' from assembly 'NReadability

### ISSUE THREAD

**Issue by squallstar** [marek-stoj/NReadability#14]
Hello,

I'm having the following issue while loading the library:

**Could not load type 'NReadability.NReadabilityWebTranscoder' from assembly 'NReadability**

I installed the lib via NuGet, and I'm using it as follows:

```
var input = new WebTranscodingInput("http://....");
var transcoder = new NReadabilityWebTranscoder();
var result = transcoder.Transcode(d);
```

But as soon as I start the compiled build, I get that error and the app stops.

Please find the attached screenshot about the error.

Could you help please? :(

Thanks
Nicholas

![Screen Shot 2013-02-28 at 16 18 35](https://f.cloud.github.com/assets/574210/206078/a0747d54-81c2-11e2-9f09-276b97817b2f.png)

**Comment by squallstar** [comment]
Could the problem be that I'm using the lib with a **Windows Phone** project?

**Comment by marek-stoj** [comment]
Yes, the lib was not tested on Windows Phone. It should be fairly easy to
port the code. Just use the full code and see what is causing the exception.

Best Regards,
Marek Stój

## 

http://www.marekstoj.com
http://www.devmedia.pl

**Comment by squallstar** [comment]
Hi Marek,
Thanks for your reply. I'll try to import the Src/Nreadability sources to project and see what happens.

Nicholas

**Comment by marek-stoj** [comment]
Great, let me know if you find out anything - I'll comment on the issue or
just fix the code on GitHub.

Best Regards,
Marek Stój

## 

http://www.marekstoj.com
http://www.devmedia.pl
